build.gradle.kts

```diff
@@ -191,6 +184,27 @@ dependencies {
   // jhipster-needle-gradle-test-dependencies
 }
 
+val profile = project.findProperty("profile") as String? ?: "default"
```

Comment:
You seems to be missing the poing that multiple build profiles can be active by default, and they should all be activated.

Reply:
This PR contains a 2nd commit that improves things: https://github.com/renanfranca/gradleapp-profiles-sample/pull/2/files

Here's how it works:
- `./gradlew build -x test --rerun-tasks -Pprofiles=local && grep "active:" build/resources/main/config/application.yml` -> `active: "local"`
- `./gradlew build -x test --rerun-tasks -Pprofiles=dev && grep "active:" build/resources/main/config/application.yml` -> `active: "dev"`
- `./gradlew build -x test --rerun-tasks && grep "active:" build/resources/main/config/application.yml` -> `active: "local"`

Reply:
Thanks @murdos ! Can I start the implementation? WDYT?

Reply:
Yes, let's try that way! 🙂

Reply:
> Yes, let's try that way! 🙂

Thank you very much, let's go! I will start tomorrow morning 👍